Eat_and_Greet/src/screens/PostEvent/index.tsx

```diff
@@ -113,32 +139,43 @@ export default function PostEvent() {
           </View>
         </View>
         <View style={postStyle.DateTimeContainer}>
-          <Text style={postStyle.LabelText}> Date & Time {"\n"}</Text>
-          <TextBox
-            placeholder="Enter Date..."
-            top={"-30%"}
-            height={"20%"}
-            width={"90%"}
-            onTextChange={setEventDate}
-          />
-          <TextBox
-            placeholder="Enter Start Time..."
-            top={"-38%"}
-            height={"20%"}
-            width={"90%"}
-            onTextChange={setStartTime}
-          />
-          <TextBox
-            placeholder="Enter End Time..."
-            top={"-46%"}
-            height={"20%"}
+          
+          <Text style={[postStyle.LabelText, {alignSelf: "center", flex: 1}]}> Date {"\n"}</Text>
+          <DateTimePicker style={{alignSelf: "center", flex: 1}}
+                    value={dateOfEvent}
+                    mode='date'
+                    onChange={date}
+                  />
+                  </View>
+          <View style={postStyle.DateTimeContainer}>
+          <Text style={[postStyle.LabelText, {alignSelf: "center", flex: 1}]}> Start Time {"\n"}</Text>
+          <DateTimePicker style={{alignSelf: "center", flex: 1}}
+                    value={startTime}
+                    mode='time'
+                    onChange={startingTime}
+                  />
+        </View>
+        <View style={postStyle.DateTimeContainer}>
+          <Text style={[postStyle.LabelText, {alignSelf: "center", flex: 1}]}> End Time {"\n"}</Text>
+          <DateTimePicker style={{alignSelf: "center", flex: 1}}
+                    value={endTime}
+                    mode='time'
+                    onChange={endingTime}
+                  />
+        </View>
+        <View style={postStyle.DescriptionContainer}>
+          <Text style={postStyle.LabelText}>Description</Text>
+          <GrowingTextbox
+            placeholder="Enter Description..."
+            height={"80%"}
             width={"90%"}
-            onTextChange={setEndTime}
-          />
+            onTextChange={setDescription}
+        />
         </View>
-      </SafeAreaView>
-      <RoundedButton name="Post Event" top="-2%" onPress={addPostToDB} />
+      </KeyboardAwareScrollView>
+      </View>
+      <RoundedButton name="Post Event" onPress={addPostToDB} />
       <Toolbar />
-    </SafeAreaView>
+    </View>
```

Comment:
Curious why you went from SafeAreaView to regular View? 
Prolly doesn't matter with scroll view anyway but curious